site/profile/manifests/core/fiberspec.pp

```diff
@@ -1,10 +1,23 @@
 # @summary
-#   Manages Avantes Fiberspectrometer USB devices.
+#   Manages Avantes Fiberspectrometer USB devices and udev configuration.
 #
 class profile::core::fiberspec {
   systemd::udev::rule { 'fiberspec.rules':
     rules => [
       "SUBSYSTEM==\"usb\", ATTR{idVendor}==\"1992\", ATTR{idProduct}==\"0667\", ACTION==\"add\", GROUP=\"saluser\", MODE=\"0664\"",
     ],
   }
+
+  file_line { 'udev_log_debug':
```

Comment:
Could we use `systemd::udev_log` here?

https://github.com/voxpupuli/puppet-systemd/blob/master/manifests/init.pp#L153-L154